exercises/nth-prime/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "nth_prime.h"
+#include <stdbool.h>
+#include <math.h>
+
+static bool is_prime(uint32_t n)
+{
+  for(uint32_t i = (uint32_t)sqrt(n); i > 1; i--) {
+    if(n % i == 0) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+uint32_t nth(uint32_t n)
+{
```

Comment:
A couple of these leading brackets are on the next line instead of the same line. Can you make these consistent within this file?